docs/initialization.rst

```diff
@@ -0,0 +1,330 @@
+The system initialization, i.e., the generation of the Initial Condition (IC), is of paramount importance, as it can significantly impact the performance and accuracy of the simulation. Several methods are offered in MoorDyn-C version 2, which can be concatenated to achieve the required accuracy at a minimum cost.
```

Comment:
In a future PR let's remove these lines (1-5). Lines 1-5 are duplicates of 14-18